build.cmd

```diff
@@ -23,7 +23,7 @@ set "__LogsDir=%__RootBinDir%\Logs"
 set __MSBCleanBuildArgs=
 set __SkipTestBuild=
 set __ToolchainMilestone=testing
-set __DotNetCliPath=
+set "__DotNetCliPath=%__ProjectDir%\Tools\dotnetcli"
 
```

Comment:
The buildtools msbuild scripts (the part that we use) do not depend on CLI today. All dependencies on CLI in our msbuild scripts are local to our repo (restore, packaging)...